src/Object/Post.php

```diff
@@ -768,6 +768,35 @@ private function getCommentBoxTemplate()
 		return $this->comment_box_template;
 	}
 
+	/**
+	 * Get default text for the comment box
+	 *
+	 * @param integer $parent_id ID of the parent item
+	 *
+	 * @return string
+	 */
+	private function getDefaultText($parent_id)
```

Comment:
I would call the `$this->getId()` directly inside this method rather than adding it as a parameter because:
- it's a private method so nobody outside will call it
- it is only used for `$this->getId()`

Reply:
Now that you are a contributor on Friendica GitHub, you can submit proper "Request changes" reviews, approvals and merge once tests have passed and at least one approval has been submitted, including your own.

Reply:
Thanks, good to know :-) :-)

Reply:
@mrpetovan are you sure that all should work? I don't see any difference yet..

Reply:
No, I'm not sure because I don't have your view in GitHub nor have I personally the rights to promote contributors. However, I can now select you in the assignment list for issues and pull requests, and I thought this list only contained contributors with merge rights. I may be mistaken though.